paper-tabs.html

```diff
@@ -587,45 +629,47 @@
           this.$.selectionBar.classList.remove('expand');
           this.$.selectionBar.classList.remove('contract');
           this._positionBar(this._pos.width, this._pos.left);
-          return;
-        }
-
-        var oldRect = old.getBoundingClientRect();
-        var oldIndex = this.items.indexOf(old);
-        var index = this.items.indexOf(tab);
-        var m = 5;
-
-        // bar animation: expand
-        this.$.selectionBar.classList.add('expand');
-
-        var moveRight = oldIndex < index;
-        var isRTL = this._isRTL;
-        if (isRTL) {
-          moveRight = !moveRight;
-        }
-
-        if (moveRight) {
-          this._positionBar(this._calcPercent(tabRect.left + tabRect.width - oldRect.left, w) - m,
-              this._left);
         } else {
-          this._positionBar(this._calcPercent(oldRect.left + oldRect.width - tabRect.left, w) - m,
-              this._calcPercent(tabOffsetLeft, w) + m);
+          var oldRect = old.getBoundingClientRect();
+          var oldIndex = this.items.indexOf(old);
+          var index = this.items.indexOf(tab);
+          var m = 5;
```

Comment:
It was over [here](https://github.com/PolymerElements/paper-tabs/pull/142/files#diff-eae51ac6909052c1dbe88871dd1695b1L596). After intense staring, I think it might be 'margin'? I'll clean up this section a bit..